torch/_functorch/aot_autograd.py

```diff
@@ -580,7 +579,15 @@ def convert(idx, x):
                 )
 
                 output_and_mutation_safe = not any(
-                    x.requires_grad for x in fw_metadata.output_info
+                    x.requires_grad
+                    # view operations preserve requires_grad even in no_grad.
+                    # Do not count aliases of inputs with requires_grad as reason to make a training graph.
```

Comment:
nit: I would mention in the comment that it AOTAutograd will perform view-replay to regenerate the view outputs at runtime, setting their `grad_fn` properly.

Otherwise, it's not clear just from the comment about why it's ok to "ignore" views that require grad when deciding that we can generate an inference graph here.

Reply:
Thanks, added.